ROI_Maker.py

```diff
@@ -8,31 +7,32 @@ def ROI_Maker(spectrum, energy,sub_regions='auto'):
     """
     Takes in a measured spectrum and identifies peaks in terms of channel number
     and energy. Uses peak_finder to localize the peak more precisely from the
-    given energy. Also returns peak and compton ROIs as well as each ROIs gross
-    area.
+    given energy. Also returns peak and compton ROIs.
     """
 
     peak_energy = peak_finder(spectrum, energy)
 
     E0 = spectrum.energy_cal[0]
     Eslope = spectrum.energy_cal[1]
-    M_counts = spectrum.data
-    energy_channel = int((peak_energy - E0) / Eslope)
+    energy_ch = int((peak_energy - E0) / Eslope)
 
     region_size = 1.3
-    compton_distance = 4
+    separation_parameter = 2
 
     # Rough estimate of FWHM.
-    fwhm = 0.05*peak_energy**0.5
-    fwhm_channel = int(region_size * (fwhm - E0) / Eslope)
-    # peak gross area
-    peak_ch = (energy_channel - fwhm_channel, energy_channel + fwhm_channel)
+    fwhm_kev = 0.05*peak_energy**0.5
+    fwhm_ch = (fwhm_kev - E0)/Eslope
 
-    left_center = energy_channel - compton_distance * fwhm_channel
-    left_ch = (left_center - fwhm_channel, left_center + fwhm_channel)
+    region_half_width = int(region_size * fwhm_ch)
+    region_separation = int(separation_parameter*region_half_width)
```

Comment:
Let's define `region_separation` in terms of `fwhm_ch` instead of `region_half_width`. So a new `separation_parameter` would be 2 * 1.3 = 2.6, which is the number of FWHM's in between the ROIs.

(As a side note, the math is cleaner if we only call `int()` once, and `region_half_width` already had `int()` in it. I think this is related to the minor issue I mentioned in #52.)